src/components/index.js

```diff
@@ -124,4 +126,6 @@ export {
   TypingIndicator,
   TypingIndicatorContainer,
   UploadProgressIndicator,
+  IconBadge,
+  IconSquare,
```

Comment:
prob want to put these in some alphabetical order but we already have it covered in the typescript stuff